python/llm/example/NPU/HF-Transformers-AutoModels/LLM/CPP_Examples/README.md

```diff
@@ -0,0 +1,75 @@
+# C++ Example of running HuggingFace `transformers` Models on Intel NPU using IPEX-LLM
+In this directory, you will find a C++ example on how to run HuggingFace `transformers` LLM models on Intel NPUs (leveraging *Intel NPU Acceleration Library*). See the table blow for verified models.
```

Comment:
a C++ example on how to run LLM models on Intel NPUs using IPEX-LLM

Reply:
update to this.